webapp/app/modules/users/users.selectors.js

```diff
@@ -0,0 +1,18 @@
+import { createSelector } from 'reselect';
+import { Map } from 'immutable';
+
+import { selectUserUid } from '../userAuth/userAuth.selectors';
+
+export const selectDomain = (state) => state.get('users', Map());
+
+export const selectUsers = createSelector(
+  selectDomain,
+  (state) => state.get('items', Map()),
```

Comment:
List() ?

Reply:
Jeśli chodzi o user'ów to nie do końca widzę jakiś profit z tego że ta lista będzie uporządkowana . Bo tak w tym momencie jest to Map.

Reply:
@wojciech-panek @mkmieciak Jeśli nie jest to jakaś wielka zbrodnia dla Immutable to tak bym zostawił.

Reply:
A czy to nie jest nawet tak, że firebase nie wspiera tablic i ci userzy są właśnie mapą? Mi to nie przeszkadza w każdym razie.